tests/at_end2end_test/lib/src/at_encryption_key_initializers.dart

```diff
@@ -23,22 +23,6 @@ class AtEncryptionKeysLoader {
 
   /// Stores the encryption keys to the local secondary key-store.
   Future<void> setEncryptionKeys(AtClient atClient, String atSign) async {
-    // Check if the encryption keys are already for the atSign.
-    // If yes, return; else set the encryption keys.
-    String? encryptionPublicKey;
-    try {
-      encryptionPublicKey =
-          await atClient.getLocalSecondary()?.getEncryptionPublicKey(atSign);
-    } on KeyNotFoundException {
-      _logger.info('Encryption keys are not found. Adding to the keystore.');
-    }
-
-    if (encryptionPublicKey.isNotNull && encryptionPublicKey != 'data:null') {
-      _logger.info(
-          'The encryption keys are already updated to keystore, returning');
-      return;
-    }
-
```

Comment:
Deleted the check above, we always want to store the encryption keys to local keystore